vnext/src-win/Libraries/Components/View/ViewPropTypes.windows.js

```diff
@@ -456,6 +456,7 @@ type ViewPropsWindows = $ReadOnly<{|
   'aria-setsize'?: ?number,
   accessibilityDescription?: ?string,
   'aria-description'?: ?string,
+  'aria-hidden'?: ?boolean,
```

Comment:
We shouldn't need to add a windows specific property here... we should be able to use the existing one from core. (accessibilityElementsHidden)